README.md

```diff
@@ -67,6 +67,7 @@ You can join our [discord server](https://discord.gg/5Xh2W87egW) for commits, ch
 - InRole by nin0dev
 - IrcColors by Grzesiek11
 - IRememberYou by zoodogood
+- ImagePreview by Creation's
```

Comment:
dont forget to increase the amount at the top 
![image](https://github.com/user-attachments/assets/9e1c3873-910b-43f5-9e3f-4d9d4afd94e6)